buf.yaml

```diff
@@ -0,0 +1,25 @@
+version: v1beta1
+build:
+  roots:
+    - integration_test/proto
+lint:
+  use:
+    - DEFAULT
+  ignore_only:
+    ENUM_VALUE_PREFIX:
+      - improbable/grpcweb/test/test.proto
+    ENUM_ZERO_VALUE_SUFFIX:
+      - improbable/grpcweb/test/test.proto
+    FIELD_LOWER_SNAKE_CASE:
+      - improbable/grpcweb/test/test.proto
+    PACKAGE_VERSION_SUFFIX:
+      - improbable/grpcweb/test/test.proto
+    RPC_REQUEST_RESPONSE_UNIQUE:
+      - improbable/grpcweb/test/test.proto
+    RPC_REQUEST_STANDARD_NAME:
+      - improbable/grpcweb/test/test.proto
+    RPC_RESPONSE_STANDARD_NAME:
+      - improbable/grpcweb/test/test.proto
```

Comment:
These ignores are based on the existing failures in this file. These are the warnings:
```
$ buf lint
integration_test/proto/improbable/grpcweb/test/test.proto:3:1:Package name "improbable.grpcweb.test" should be suffixed with a correctly formed version, such as "improbable.grpcweb.test.v1".
integration_test/proto/improbable/grpcweb/test/test.proto:10:5:Enum value name "NONE" should be prefixed with "FAILURE_TYPE_".
integration_test/proto/improbable/grpcweb/test/test.proto:10:5:Enum zero value name "NONE" should be suffixed with "_UNSPECIFIED".
integration_test/proto/improbable/grpcweb/test/test.proto:11:5:Enum value name "CODE" should be prefixed with "FAILURE_TYPE_".
integration_test/proto/improbable/grpcweb/test/test.proto:13:5:Enum value name "CODE_UNICODE" should be prefixed with "FAILURE_TYPE_".
integration_test/proto/improbable/grpcweb/test/test.proto:28:10:Field name "Value" should be lower_snake_case, such as "value".
integration_test/proto/improbable/grpcweb/test/test.proto:39:3:".google.protobuf.Empty" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:39:3:".improbable.grpcweb.test.PingResponse" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:39:17:RPC request type "Empty" should be named "PingEmptyRequest" or "TestServicePingEmptyRequest".
integration_test/proto/improbable/grpcweb/test/test.proto:39:49:RPC response type "PingResponse" should be named "PingEmptyResponse" or "TestServicePingEmptyResponse".
integration_test/proto/improbable/grpcweb/test/test.proto:41:3:".improbable.grpcweb.test.PingRequest" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:41:3:".improbable.grpcweb.test.PingResponse" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:43:3:".google.protobuf.Empty" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:43:3:".improbable.grpcweb.test.PingRequest" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:43:17:RPC request type "PingRequest" should be named "PingErrorRequest" or "TestServicePingErrorRequest".
integration_test/proto/improbable/grpcweb/test/test.proto:43:39:RPC response type "Empty" should be named "PingErrorResponse" or "TestServicePingErrorResponse".
integration_test/proto/improbable/grpcweb/test/test.proto:45:3:".improbable.grpcweb.test.PingRequest" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:45:3:".improbable.grpcweb.test.PingResponse" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:45:16:RPC request type "PingRequest" should be named "PingListRequest" or "TestServicePingListRequest".
integration_test/proto/improbable/grpcweb/test/test.proto:45:45:RPC response type "PingResponse" should be named "PingListResponse" or "TestServicePingListResponse".
integration_test/proto/improbable/grpcweb/test/test.proto:47:3:".improbable.grpcweb.test.PingRequest" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:47:3:".improbable.grpcweb.test.PingResponse" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:47:27:RPC request type "PingRequest" should be named "PingPongBidiRequest" or "TestServicePingPongBidiRequest".
integration_test/proto/improbable/grpcweb/test/test.proto:47:56:RPC response type "PingResponse" should be named "PingPongBidiResponse" or "TestServicePingPongBidiResponse".
integration_test/proto/improbable/grpcweb/test/test.proto:49:3:".improbable.grpcweb.test.PingRequest" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:49:3:".improbable.grpcweb.test.PingResponse" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:49:25:RPC request type "PingRequest" should be named "PingStreamRequest" or "TestServicePingStreamRequest".
integration_test/proto/improbable/grpcweb/test/test.proto:49:47:RPC response type "PingResponse" should be named "PingStreamResponse" or "TestServicePingStreamResponse".
integration_test/proto/improbable/grpcweb/test/test.proto:51:3:RPC "Echo" has the same type ".improbable.grpcweb.test.TextMessage" for the request and response.
integration_test/proto/improbable/grpcweb/test/test.proto:51:12:RPC request type "TextMessage" should be named "EchoRequest" or "TestServiceEchoRequest".
integration_test/proto/improbable/grpcweb/test/test.proto:51:34:RPC response type "TextMessage" should be named "EchoResponse" or "TestServiceEchoResponse".
integration_test/proto/improbable/grpcweb/test/test.proto:67:3:".google.protobuf.Empty" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:67:54:RPC response type "Empty" should be named "ContinueStreamResponse" or "TestUtilServiceContinueStreamResponse".
integration_test/proto/improbable/grpcweb/test/test.proto:73:3:".improbable.grpcweb.test.PingRequest" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:73:3:".improbable.grpcweb.test.PingResponse" is used as the request or response type for multiple RPCs.
integration_test/proto/improbable/grpcweb/test/test.proto:73:19:RPC request type "PingRequest" should be named "NonExistantRequest" or "FailServiceNonExistantRequest".
integration_test/proto/improbable/grpcweb/test/test.proto:73:41:RPC response type "PingResponse" should be named "NonExistantResponse" or "FailServiceNonExistantResponse".
```

Reply:
I didn't want to fix any warnings in this PR to make the impact as small as possible.